pkg/virt-launcher/virtwrap/live-migration-source.go

```diff
@@ -409,13 +408,7 @@ func (l *LibvirtDomainManager) startMigration(vmi *v1.VirtualMachineInstance, op
 		return nil
 	}
 
-	err = l.asyncMigrate(vmi, options)
-	if err != nil {
-		log.Log.Object(vmi).Reason(err).Error(liveMigrationFailed)
-		l.setMigrationResult(vmi, true, fmt.Sprintf("%v", err), "")
-		return err
-	}
-
+	l.asyncMigrate(vmi, options)
```

Comment:
IMHO we could do even better. `go l.migrate(vmi, options)` would be more idiomatic

Reply:
Makes sense, thanks! Fixed.